accepted/2023/owner-author-pmui.md

```diff
@@ -0,0 +1,78 @@
+In the case of `nuget.org`, also provide a hyperlink to the owner's profile page (ie, `https://www.nuget.org/profiles/<owner name>`).
+A Trusted Owner is defined as an owner that's indicated by a package source's [Search API](https://learn.microsoft.com/nuget/api/search-query-service-resource#search-result), as opposed to the field in `.nuspec`.
+
+## Motivation 
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+
+Customers need a clear understanding of who created a NuGet package in order to trust that package.
+The term "owner" has become overloaded as NuGet has evolved, which has caused confusion. 
+The "author" metadata is also misleading as it's a free-form text field which doesn't imply any credentials.
+
+## Explanation
+
+### Functional explanation
+
+<!-- Explain the proposal as if it were already implemented and you're teaching it to another person. -->
+<!-- Introduce new concepts, functional designs with real life examples, and low-fidelity mockups or  pseudocode to show how this proposal would look. -->
+
+![Owners from NuGet.org are shown beside each package ID with a hyperlink and a NuGet Package Icon](../../meta/resources/OwnerAuthor/ownerNuGetOrgLinks.png)
+
+### Technical explanation
+
+<!-- Explain the proposal in sufficient detail with implementation details, interaction models, and clarification of corner cases. -->
+
+Currently, `Owners` JSON parsed from the Search API is immediately converted into a comma-delimited string of owners.
+To avoid breaking any clients depending on this, `Owners` will continue being a pre-processed string with comma-delimited owners.
+A new property, `OwnersEnumerable` (naming TBD) will be added to [IPackageSearchMetadata](https://github.com/NuGet/NuGet.Client/blob/dc2fadacd311646754ce6b062489b403338972aa/src/NuGet.Core/NuGet.Protocol/Model/IPackageSearchMetadata.cs) and [PackageSearchMetadataContextInfo](https://github.com/NuGet/NuGet.Client/blob/dc2fadacd311646754ce6b062489b403338972aa/src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/ContextInfos/PackageSearchMetadataContextInfo.cs).
+The enumerable owners will allow displaying custom UI for each owner.
+
+Building a profile URL will be done by applying the pattern for each owner: `www.nuget.org/profiles/<owner name>`.
+
+TBD: For non-nuget.org sources... 
+- don't link to nuget.org profile? link to some other profile?
+- Still show the NuGet Package icon.
+
+## Drawbacks
+
+<!-- Why should we not do this? -->
+
+## Rationale and alternatives
+
+<!-- Why is this the best design compared to other designs? -->
+<!-- What other designs have been considered and why weren't they chosen? -->
+<!-- What is the impact of not doing this? -->
+
+Using `by:` in the same manner as `www.nuget.org`, we could start letting this imply Owner, and introduce a new term for Author.
```

Comment:
Personally I think what you have in the main design is find. We never stated "authors" before so I don't know if many users have a strong expectation of what this value is (owners vs authors) meaning distinguishing the two might not add a lot of value. Just my opinion.